<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1542</width>
    <height>849</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow {
    background-color: #000000;  /* 黑色背景 */
    color: #FFFFFF;  /* 白色文字 */
}

</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">QWidget {
    background-color: #000000;  /* 黑色背景 */
    color: #FFFFFF;  /* 白色文字 */
}</string>
   </property>
   <widget class="QPushButton" name="helpButtonLeft">
    <property name="geometry">
     <rect>
      <x>1250</x>
      <y>610</y>
      <width>291</width>
      <height>241</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: red; /* 按钮背景颜色 */
    border-radius: 0px; /* 圆角半径，确保是圆形 */
    width: 100px; /* 按钮宽度 */
    height: 100px; /* 按钮高度 */
    color: white; /* 字体颜色 */
    border: 2px solid #00FF00;  /* 绿色边框 */
    font-family: &quot;Microsoft YaHei&quot;; /* 设置字体为黑体 */
    font-weight: bold; /* 字体加粗 */
    font-size: 16px; /* 字体大小 */
}
</string>
    </property>
    <property name="text">
     <string>紧急求助</string>
    </property>
   </widget>
   <widget class="QPushButton" name="askLLMButton">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>660</y>
      <width>991</width>
      <height>121</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #000000;  /* 黑色背景 */
    border: 2px solid #00FF00;  /* 绿色边框 */
    color: #FFFFFF;  /* 白色文字 */
}</string>
    </property>
    <property name="text">
     <string>语音助手</string>
    </property>
   </widget>
   <widget class="QPushButton" name="helpButtonRight">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>610</y>
      <width>291</width>
      <height>241</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: red; /* 按钮背景颜色 */
    border-radius: 0px; /* 圆角半径，确保是圆形 */
    width: 100px; /* 按钮宽度 */
    height: 100px; /* 按钮高度 */
    color: white; /* 字体颜色 */
    border: 2px solid #00FF00;  /* 绿色边框 */
    font-family: &quot;Microsoft YaHei&quot;; /* 设置字体为黑体 */
    font-weight: bold; /* 字体加粗 */
    font-size: 16px; /* 字体大小 */
}
</string>
    </property>
    <property name="text">
     <string>紧急求助</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="carStatusBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>451</width>
      <height>411</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QGraphicsView {
    background-image: url(../oncarDesktop/carmodel.jpg); /* 使用本地图片路径 */
    background-repeat: no-repeat; /* 不重复图片 */
    background-position: center; /* 图片居中 */
    background-size: cover; /* 图片按比例填充整个视图 */
}</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="userWidget">
    <property name="geometry">
     <rect>
      <x>1330</x>
      <y>10</y>
      <width>201</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QTextBrowser {
    background-color: #000000;  /* 黑色背景 */
    color: #FFFFFF;  /* 白色文字 */
}</string>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="avatarWidget">
    <property name="geometry">
     <rect>
      <x>1260</x>
      <y>10</y>
      <width>71</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QGraphicsView {
    background-color: #000000;  /* 黑色背景 */
    color: #FFFFFF;  /* 白色文字 */
}</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="timeWidget">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>20</y>
      <width>531</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QTextBrowser {
    background-color: #000000;  /* 黑色背景 */
    color: #FFFFFF;  /* 白色文字 */
    font-size:48px;  /* 设置字体大小为32号 */
    font-weight: bold;  /* 设置文字加粗 */
    font-family: &quot;Segoe UI&quot;, &quot;Arial&quot;, sans-serif;  /* 设置字体，优先使用漂亮的 &quot;Segoe UI&quot; */
    text-align: center;  /* 设置文字居中 */
}</string>
    </property>
   </widget>
   <widget class="QPushButton" name="navigateHomeButton">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>610</y>
      <width>161</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #000000;  /* 黑色背景 */
    border: 2px solid #00FF00;  /* 绿色边框 */
    color: #FFFFFF;  /* 白色文字 */
}</string>
    </property>
    <property name="text">
     <string>回家</string>
    </property>
   </widget>
   <widget class="QPushButton" name="navigateButton">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>610</y>
      <width>391</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #000000;  /* 黑色背景 */
    border: 2px solid #00FF00;  /* 绿色边框 */
    color: #FFFFFF;  /* 白色文字 */</string>
    </property>
    <property name="text">
     <string>导航</string>
    </property>
   </widget>
   <widget class="QPushButton" name="navigateWorkButton">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>610</y>
      <width>161</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #000000;  /* 黑色背景 */
    border: 2px solid #00FF00;  /* 绿色边框 */
    color: #FFFFFF;  /* 白色文字 */
}</string>
    </property>
    <property name="text">
     <string>上班</string>
    </property>
   </widget>
   <widget class="QScrollArea" name="appScrollArea">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>140</y>
      <width>1051</width>
      <height>411</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QScrollArea{
	background-color: #000000;  /* 黑色背景 */
}</string>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="appScrollAreaContent">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1049</width>
       <height>409</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QWidget {
    background-color: #000000;  /* 黑色背景 */
    color: #FFFFFF;  /* 白色文字 */
}</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="settingsButton">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>780</y>
      <width>481</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #000000;  /* 黑色背景 */
    border: 2px solid #00FF00;  /* 绿色边框 */
    color: #FFFFFF;  /* 白色文字 */
}</string>
    </property>
    <property name="text">
     <string>设置</string>
    </property>
   </widget>
   <widget class="QPushButton" name="userButton">
    <property name="geometry">
     <rect>
      <x>1250</x>
      <y>10</y>
      <width>271</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
   background: transparent; /* 透明背景 */
    border: 2px solid #00FF00;  /* 绿色边框 */
    color: #FFFFFF;  /* 白色文字 */
    font-size: 16px; /* 字体大小 */
    font-weight: bold; /* 字体加粗 */
    font-family: &quot;Arial&quot;, sans-serif; /* 字体 */
    padding: 10px 20px; /* 内边距 */
}

QPushButton:hover {
    background-color: rgba(0, 0, 0, 0.1); /* 悬停时的淡黑色背景（半透明） */
    color: blue; /* 悬停时文字颜色 */
}

QPushButton:pressed {
    background-color: rgba(0, 0, 0, 0.2); /* 按下时的较深黑色背景（半透明） */
    color: red; /* 按下时文字颜色 */
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="mediaButton">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>780</y>
      <width>481</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #000000;  /* 黑色背景 */
    border: 2px solid #00FF00;  /* 绿色边框 */
    color: #FFFFFF;  /* 白色文字 */</string>
    </property>
    <property name="text">
     <string>媒体</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="systemName">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>481</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QTextBrowser {
    background-color: #000000;  /* 黑色背景 */
    color: #FFFFFF;  /* 白色文字 */
    font-size:24px;  /* 设置字体大小为32号 */
    font-weight: bold;  /* 设置文字加粗 */
    font-family: &quot;Segoe UI&quot;, &quot;Arial&quot;, sans-serif;  /* 设置字体，优先使用漂亮的 &quot;Segoe UI&quot; */
    text-align: center;  /* 设置文字居中 */
}</string>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Segoe UI','Arial','sans-serif'; font-size:24px; font-weight:600; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <zorder>appScrollArea</zorder>
   <zorder>askLLMButton</zorder>
   <zorder>helpButtonLeft</zorder>
   <zorder>helpButtonRight</zorder>
   <zorder>carStatusBox</zorder>
   <zorder>userWidget</zorder>
   <zorder>avatarWidget</zorder>
   <zorder>timeWidget</zorder>
   <zorder>navigateHomeButton</zorder>
   <zorder>navigateButton</zorder>
   <zorder>navigateWorkButton</zorder>
   <zorder>settingsButton</zorder>
   <zorder>userButton</zorder>
   <zorder>mediaButton</zorder>
   <zorder>systemName</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
